src/main/java/net/dv8tion/jda/api/interactions/commands/build/SlashCommandData.java

```diff
@@ -332,7 +337,10 @@ static SlashCommandData fromCommand(@Nonnull Command command)
             throw new IllegalArgumentException("Cannot convert command of type " + command.getType() + " to SlashCommandData!");
 
         CommandDataImpl data = new CommandDataImpl(command.getName(), command.getDescription());
-        data.setDefaultEnabled(command.isDefaultEnabled());
+        if (!command.isCommandEnabledInDMs())
+            data.setCommandEnabledInDMs(false);
+        if (command.getDefaultPermissionsRaw() != -1)
```

Comment:
I'm uncertain whether -1 is a good sentinel value to use, maybe a boolean is better suited